### PR TITLE
add exercise 14

### PR DESCRIPTION
**Pull request by barrantes-sofiaanne**
Pleaes review my file. Thank You!

### CODE REVIEW

File: barrantes_sofiaanne/countries.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Countries of the World</title>
+    <link rel="stylesheet" href="sample.css">
+</head>
+<body>
+    <div class="container">
+        <h1>Countries of the World</h1>
+        <input type="text" id="countryInput" placeholder="Enter country name...">
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/countries.html]: exceed 80 cols

File: barrantes_sofiaanne/index.html [hunk 2]
```diff
@@ -7,6 +7,10 @@
     <title>Barrantes, Sofia Anne</title>
 </head>
 <body>
+    <div class="header">
+        <button>Countries</button>
+        <a href="countries.html"></a>
+    </div>
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.html]: can you add or make the button bigger cause at first glance it's so hard to find.

Comment (MentokAlmera) [reply barrantes_sofiaanne/index.html]: also please check since the href is not working.

File: barrantes_sofiaanne/index.js [hunk 3]
```diff
@@ -62,3 +62,61 @@ document.addEventListener("DOMContentLoaded", function() {
 
   displayComments();
 });
+
+  function searchCountry() {
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.js]: remove indention

File: barrantes_sofiaanne/index.js [hunk 4]
```diff
@@ -62,3 +62,61 @@ document.addEventListener("DOMContentLoaded", function() {
...
+  let countryName = document.getElementById('countryInput').value.trim();
+  if (!countryName) {
+  document.getElementById('countryDetails').innerHTML = '<p>Please enter a country name.</p>';
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.js]: exceed 80 cols

File: barrantes_sofiaanne/index.js [hunk 5]
```diff
@@ -62,3 +62,61 @@ document.addEventListener("DOMContentLoaded", function() {
...
+  document.getElementById('sameRegionCountries').innerHTML = '';
+  return;
+  }
+  
+  fetch('https://restcountries.com/v3.1/name/' + countryName)
+  .then(function(response) {
+  if (!response.ok) {
+   throw new Error('Country not found');
+  }
+  return response.text();
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.js]: indent

File: barrantes_sofiaanne/index.js [hunk 6]
```diff
@@ -62,3 +62,61 @@ document.addEventListener("DOMContentLoaded", function() {
...
+  let country = JSON.parse(countryData)[0];
+  let details = `
+   <h2>Country Details - ${country.name.common}</h2>
+   <img src="${country.flags.svg}" alt="Flag of ${country.name.common}" width="100">
+   <p><strong>Area:</strong> ${country.area ? country.area.toLocaleString() + ' square kilometers' : 'N/A'}</p>
+   <p><strong>Languages:</strong> ${country.languages ? Object.values(country.languages).join(', ') : 'N/A'}</p>
+   <p><strong>Subregion:</strong> ${country.subregion ? country.subregion : 'N/A'}</p>
+   <p><strong>Capital:</strong> ${country.capital ? country.capital[0] : 'N/A'}</p>
+   <p><strong>Timezones:</strong> ${country.timezones ? country.timezones.join(', ') : 'N/A'}</p>
+  `;
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.js]: exceed 80 cols

File: barrantes_sofiaanne/index.js [hunk 7]
```diff
@@ -62,3 +62,61 @@ document.addEventListener("DOMContentLoaded", function() {
...
+  document.getElementById('countryDetails').innerHTML = details;
+  
+  return fetch('https://restcountries.com/v3.1/region/' + country.region);
+  })
+  .then(function(response) {
+  if (!response.ok) {
+   throw new Error('Region not found');
+  }
+  return response.text();
+  })
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.js]: check indention

Comment (MentokAlmera) [reply barrantes_sofiaanne/index.js]: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR0-Mlshd3ywENZTBvqD9pN1RHaAvWpVA50RX5HZgFTEDeYzhmtZh1a1r8k#slide=id.g29e6ce31160_0_11

File: barrantes_sofiaanne/index.js [hunk 8]
```diff
@@ -62,3 +62,61 @@ document.addEventListener("DOMContentLoaded", function() {
...
+  document.getElementById('sameRegionCountries').innerHTML = `
+   <h2>Countries in the Same Region (${region})</h2>
+   <div class="country-list">${sameRegionCountriesList}</div>
+  `;
+  })
+  .catch(function(error) {
+  console.error('Error fetching data:', error);
+  document.getElementById('countryDetails').innerHTML = '<p>An error occurred: ' + error.message + '</p>';
+  document.getElementById('sameRegionCountries').innerHTML = '';
+  });
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.js]: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR2mb5Qre33CFfRuTtpvAN1dTOpbICXFQJSlA9Pa6x8KcjHwHelDlK8iGzQ#slide=id.g29e6ce31160_0_27

**Review by MentokAlmera — CHANGES_REQUESTED** [review]
The countrie button is not working then please fix all your indention for the countries js. :)

File: barrantes_sofiaanne/index.js [hunk 9]
```diff
@@ -63,42 +63,49 @@ document.addEventListener("DOMContentLoaded", function() {
...
-  return response.text();
-  })
+    if (!response.ok) {
+      throw new Error('Country not found');
+    }
+    return response.text();
+    })
   .then(function(countryData) {
   let country = JSON.parse(countryData)[0];
   let details = `
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.js]: from here fix all indention

**Review by MentokAlmera — CHANGES_REQUESTED** [review]
review your indention

**Review by MentokAlmera — APPROVED** [review]
All goods na. You can merge it now. <3

**Comment by barrantes-sofiaanne** [comment]
Thank you po

**Pull request merged**